torch/quantization/_numeric_suite.py

```diff
@@ -218,7 +238,9 @@ def prepare_model_with_stubs(float_module, q_module, module_swap_list, Logger):
         q_module._modules[key] = value
 
 
-def compare_model_stub(float_model, q_model, module_swap_list, data, Logger=Logger):
+def compare_model_stub(
+    float_model, q_model, module_swap_list, data, Logger=RecordingLogger
+):
     r"""Returns a dict with key corresponding to module names and each entry being
     a dictionary with two keys 'float' and 'quantized', containing the output
```

Comment:
Please add a more descriptive comment stating more details on what this function does. For example, look at the documentation for https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L214, so that the user gets more information. An example code snippet as part of the comments is very useful. You can reuse some material from the tutorial here too.